### DISCUSSION TITLE
How do I triger hover event on youtube?

### DISCUSSION THREAD

**Discussion by glyh** [gdh1995/vimium-c#579] [Q&A] [answered]
As title, I tried this:
```
map c LinkHints.activateHover
```

It would not pop up the preview window, like below:
![image](https://user-images.githubusercontent.com/12870742/158941454-f22b3e39-2e8c-4c4d-bf93-89162905dc70.png)

It just focus on that element.

**Comment by gdh1995** [comment]
I can't visit it in recent days, so I can't do tests... Maybe in some days it will be accessible again.

**Comment by gdh1995** [comment, marked as answer]
Um, this problem can not be fixed, because: Youtube uses JS code to show such previewing panel, but Vimium C can not know how to trigger its JS code.

Extensions like Vimium C can only "simulate" the `hover` effects. As said in https://github.com/gdh1995/vimium-c/issues/86#issuecomment-549799261, the simulation has many limits and it often fails in complicated big web pages with rich customized functionalities.
